### PR TITLE
CIV-11485 confirmation business

### PR DESCRIPTION
**Pull request by GorilaNaranja**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-11485

### Change description ###
Claimant Intent – Confirmation screens : Reject the defendant plan + Defendant = ORG or LTD Company

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/modules/i18n/locales/cy.json [hunk 1]
```diff
@@ -928,6 +928,14 @@
     },
     "CLAIMANT_RESPONSE_CONFIRMATION": {
       "CLAIM_NUMBER": "Your claim number:",
+      "COURT_BELIEVES_CAN_AFFORD": {
+        "TITLE": "If the court believes the defendant can afford your plan",
+        "CCJ_WILL_BE_ISSUED": "A County Court Judgement (CCJ) will be issued. This requires the defendant to pay according to the terms of the plan."
```
Comment (paul-pearson) [inline src/main/modules/i18n/locales/cy.json:933]: Typo - should  be "Judgment".

File: src/main/modules/i18n/locales/en.json [hunk 2]
```diff
@@ -928,6 +928,14 @@
     },
     "CLAIMANT_RESPONSE_CONFIRMATION": {
       "CLAIM_NUMBER": "Your claim number:",
+      "COURT_BELIEVES_CAN_AFFORD": {
+        "TITLE": "If the court believes the defendant can afford your plan",
+        "CCJ_WILL_BE_ISSUED": "A County Court Judgement (CCJ) will be issued. This requires the defendant to pay according to the terms of the plan."
```
Comment (paul-pearson) [inline src/main/modules/i18n/locales/en.json:933]: As above - "Judgment".

**Review by paul-pearson — CHANGES_REQUESTED** [review]

File: src/main/modules/i18n/locales/en.json (hunk 2 above)
Comment (GorilaNaranja) [reply src/main/modules/i18n/locales/en.json]: Fixed

File: src/main/modules/i18n/locales/cy.json (hunk 1 above)
Comment (GorilaNaranja) [reply src/main/modules/i18n/locales/cy.json]: Fixed

**Review by paul-pearson — APPROVED** [review]

**Review by portizdev — APPROVED** [review]

**Review by laarmada — DISMISSED** [review]

**Pull request merged**